akka-remote/src/main/scala/akka/remote/Endpoint.scala

```diff
@@ -556,7 +556,8 @@ private[remote] class EndpointWriter(
   var lastAck: Option[Ack] = None
 
   override val supervisorStrategy = OneForOneStrategy(loggingEnabled = false) {
-    case NonFatal(e) ⇒ publishAndThrow(e, Logging.ErrorLevel)
+    case e: ShutDownAssociation ⇒ publishAndThrow(e, Logging.WarningLevel)
```

Comment:
It is even just info? It happens on leaving, and that isn't necessarily something bad.

Reply:
@johanandren it is a good point. Thanks. Updated.